src/main/java/com/mysticalchemy/recipe/PotionIngredientRecipe.java

```diff
@@ -71,6 +71,10 @@ public RecipeType<?> getType() {
 	public HashMap<MobEffect,Float> getEffects(){		
 		return effects;
 	}
+
+	public void setEffects(HashMap<MobEffect,Float> effects) {
```

Comment:
Recipes should be immutable once parsed and loaded; adjustments should only be done at the event level.